public/css/style.css

```diff
@@ -81,4 +81,21 @@ a.list-group-item.active small {
 div.list-group {
   height: 300px;
   overflow: auto;
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
+
+.progress-circle {
```

Comment:
did you run the csslint for this modification ?

Reply:
Fixed.